README.md

```diff
@@ -27,3 +27,17 @@ To install these dependencies, run the command:
 ## Config file
 
 Create a copy of the `config.ini.example` file in the `build` directory called `config.ini` and update any necessary settings.
+
+## Building via Command Line
+
```

Comment:
I think it would valuable to have a script that can execute all of these commands together. 
I think we should also mention the different variants of `make`, such as `make check` and `make qmake_all` in this readme.